community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -1615,6 +1657,67 @@ else if ( txState.nodeIsAddedInThisTx( id ) )
         return storeLayer.nodeExists( id );
     }
 
+    @Override
+    public PrimitiveIntSet relationshipTypes( KernelStatement statement, NodeItem node )
```

Comment:
I think having PrimitiveIntSet for relationship types here is OK though, because it's generally not a nearly as frequently called operation as getting relationships.